packages/contracts/contracts/L1/L2OutputOracle.sol

```diff
@@ -62,12 +62,17 @@ contract L2OutputOracle is Ownable {
         uint256 _startingBlockTimestamp,
         address sequencer
     ) {
-        submissionInterval = _submissionInterval;
-        l2BlockTime = _l2BlockTime;
+        require(
+            _submissionInterval % _l2BlockTime == 0,
+            "Submission Interval must be a multiple of L2 Block Time"
+        );
```

Comment:
This prevents a situation where some outputs would be useless zero values, ie. given:

```
_submissionInterval = 2
_l2BlockTime = 5
```
Then the outputs with `timestamp` of 5, 15, 25, etc would not have corresponding l2 block.

Reply:
I chose to use a string here instead of a custom error, unlike the other errors in this contract, because this one is in the constructor, so I didn't want to add clutter with another `error` declaration at the top of this contract.